VMEncryption/main/handle.py

```diff
@@ -1582,15 +1582,17 @@ def daemon_encrypt():
 
         if (((distro_name == 'redhat' and distro_version == '7.3') or
              (distro_name == 'redhat' and distro_version == '7.4') or
-             (distro_name == 'redhat' and distro_version == '7.5')) and
+             (distro_name == 'redhat' and distro_version == '7.5') or
+             (distro_name == 'redhat' and distro_version == '7.6')) and
             (disk_util.is_os_disk_lvm() or os.path.exists('/volumes.lvm'))):
             from oscrypto.rhel_72_lvm import RHEL72LVMEncryptionStateMachine
             os_encryption = RHEL72LVMEncryptionStateMachine(hutil=hutil,
                                                             distro_patcher=DistroPatcher,
                                                             logger=logger,
                                                             encryption_environment=encryption_environment)
         elif (((distro_name == 'centos' and distro_version == '7.3.1611') or
-               (distro_name == 'centos' and distro_version.startswith('7.4'))) and
+               (distro_name == 'centos' and distro_version.startswith('7.4')) or
+               (distro_name == 'centos' and distro_version.startswith('7.5'))) and
```

Comment:
Please confirm this is tested against CentOS 7.5 LVM image outside of master prior to merge.

Reply:
checked. CentOS 7.5 is working (unmounted oldroot successfully and entered "OS Encryption Started" state, still waiting for test to finish).